flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/catalog/factory/JdbcCatalogFactory.java

```diff
@@ -66,40 +65,15 @@ public String factoryIdentifier() {
 
     @Override
     public Catalog createCatalog(Context context) {
-        final Configuration configuration = Configuration.fromMap(context.getOptions());
-        validateConfiguration(configuration);
+        final FactoryUtil.CatalogFactoryHelper helper =
```

Comment:
Can we add a test for the helper in FactoryUtilTest?

Reply:
Sure, I'll add one good/bad case tests. The called methods themselves are already tested.